src/package.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
i've noticed this from IntelliJ (I think), and I'm not sure we want it to go in or not.  @jlantz do you know?